test/test-utils.js

```diff
@@ -139,4 +139,48 @@ export class TestUtils {
 		return new TestableBlob(data, mimeType, size);
 	}
 
+	/**
+	 * Fires a new {@see TouchEvent} on the specified eventSource-element. If TouchEvent is not supported,
+	 * MouseEvents is used instead.This happens currently in firefox browsers, due to the fact that FirefoxHeadless
+	 * does not provide support for TouchEvents for now
+	 *
```

Comment:
Fires a new {@see TouchEvent} on the specified event source element. If TouchEvents are not supported,
MouseEvents are used instead. This currently affects Firefox only, due to the fact that FirefoxHeadless
does not provide support for TouchEvents for now.